Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Playwright/PlaywrightDriver.cs

```diff
@@ -223,6 +226,10 @@ public bool IsActionSupported(Act act, out string message)
             {
                 return true;
             }
+            else if (act is ActVisualTesting)
```

Comment:
Merge this condition with if on line 178, also for if on 225

Reply:
handled